### PR TITLE
WIP: Added legacy php versions

### PR DESCRIPTION
**Pull request by DracoBlue**

### CODE REVIEW

**Comment by OskarStark** [comment]
Shall we close this one?

**Comment by DracoBlue** [comment]
Nope. Maybe I will finish it once I have time spare.

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +21,10 @@ env:
   - PHP_VERSION=5.6.30
   - PHP_VERSION=5.6.31
   - PHP_VERSION=5.5.38
+  - PHP_VERSION=5.4.45
```
Comment (OskarStark) [inline .travis.yml]: 5.4. is already supported

File: .travis.yml [hunk 2]
```diff
@@ -21,6 +21,10 @@ env:
...
+  - PHP_VERSION=5.3.29
```
Comment (OskarStark) [inline .travis.yml]: this versions need their own `release/*` branch from now

**Comment by DracoBlue** [comment]
Closed. New PR if it's really needed.

**Pull request closed without merging**